packages/core/strapi/lib/middlewares/security.js

```diff
@@ -27,4 +27,19 @@ const defaults = {
 /**
  * @type {import('./').MiddlewareFactory}
  */
-module.exports = config => helmet(defaultsDeep(defaults, config));
+module.exports = config => (ctx, next) => {
+  let helmetConfig = defaultsDeep(defaults, config);
+
+  if (ctx.method === 'GET' && ['/graphql', '/documentation'].includes(ctx.path)) {
```

Comment:
Could have been useful to use `strapi.plugin('graphql').config('endpoint', '/graphql')` instead of `/graphql` here, what do you think? (I guess we can pass it through the config?)

Reply:
the plugin might not be installed so would need to use config.get('plugin.graphql.endpoint') instead. But we  should be disabling it in another way from the plugin instead at some point.